angular-material/angular-material.d.ts

```diff
@@ -73,6 +73,12 @@ declare namespace angular.material {
         initialValue(initialValue: string): IPromptDialog;
     }
 
+    interface IColorService {
+        applyThemeColors(element:Element|JQuery, colorExpression: Object): void;
+        getThemeColor(expression: Object): string;
```

Comment:
The documentation you provided seems to indicate that `expression` should be a `string`.